src/components/common/App/AppCheckbox.vue

```diff
@@ -0,0 +1,52 @@
+<script>
+import { defineComponent } from 'vue';
+export default defineComponent({
+  props: {
+    labelText: {
+      type: String,
+      required: true,
+    },
+    labelLink: {
+      text: {
+        type: String,
+      },
+      to: {
+        type: Object
+      }
+    },
```

Comment:
That's an invalid prop definition, best you can do here (without TS) is:
```suggestion
    labelLink: {
      type: Object
    },
```